docs/docsite/rst/dev_guide/testing_integration_legacy.rst

```diff
@@ -0,0 +1,73 @@
+*******************************************
+Testing using the Legacy Integration system
+*******************************************
+
+.. contents:: Topics
+
+This page details how to run the integration tests that haven't been ported to the new ``ansible-test`` framework.
+
+The following areas are still tested using the legacy ``make tests`` command:
+
+* amazon
+* azure
+* cloudflare
+* cloudscale
+* cloudstack
+* consul
+* exoscale
+* gce
+* jenkins
+* rackspace
+
+Over time the above list will be reduced as tests are ported to the ``ansible-test`` framework.
+
+
+Running Cloud Tests
+====================
+
+Cloud tests exercise capabilities of cloud modules (e.g. ec2_key).  These are
+not 'tests run in the cloud' so much as tests that leverage the cloud modules
+and are organized by cloud provider.
+
+In order to run cloud tests, you must provide access credentials in a file
+named ``credentials.yml``.  A sample credentials file named
+``credentials.template`` is available for syntax help.
+
+
```

Comment:
You could maybe add something along the lines of:

Some AWS tests may use environment variables. It is recommended to either unset any AWS environment variables (AWS_DEFAULT_PROFILE, AWS_SECRET_ACCESS_KEY, etc) or be sure that the environment variables match the credentials provided in credentials.yml to ensure the tests run with consistency to their full capability on the expected account . See http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html for information on creating a profile. 

Subsets of tests may be run by #commenting out unnecessary roles in the appropriate playbook (such as test/integration/amazon.yml).